pkg/vfs/vfs_unix.go

```diff
@@ -193,8 +186,14 @@ func (v *VFS) SetAttr(ctx Context, ino Ino, set int, fh uint64, mode, uid, gid u
 			v.writer.UpdateMtime(ino, time.Now())
 		}
 	}
-
-	err = v.Meta.SetAttr(ctx, ino, uint16(set), 0, attr)
+	if set&meta.SetAttrSize != 0 {
+		err = v.Truncate(ctx, ino, int64(size), fh, attr)
+		if err != 0 {
+			return
+		}
+	} else {
```

Comment:
Can setattr update size and mtime in the same time?

Reply:
<img width="1210" alt="image" src="https://github.com/user-attachments/assets/c21fc112-5f13-4eb8-a2dc-53c10cd16a3e" />
Mtime and ctime are updated in the implementation of doTruncate.